### PR TITLE
pin docutils, add tests, maintainer

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Notes:
- [x] adds the `docutils<0.17` pin
- [x] adds self as maintainer :wave: 
- [x] adds `pip check`
  - _probably_ would have caught the `docutils` change 
- [x] downloads the source tarball
  - this _will_ be automatically updated by the bot 
- [x] runs the tests from the source tarball, but against the as-installed package
  - as the tests deps are _nontrivial_, this does make the build go from ~4 minutes to ~7 minutes, but would have definitely caught the docutils change
- [x] sets a coverage threshold to `73`
  - this will _not_ somehow be updated by the bot, and primarily helps to catch if a new (optional) test dependency is added (or something happens per-platform)

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/pydata-sphinx-theme ready for review!

**Comment by jorisvandenbossche** [comment]
Unfortunately, the bot already merged https://github.com/conda-forge/pydata-sphinx-theme-feedstock/pull/15 .. (that's one of the disadvantages of automerge of course, when dependencies change ..)

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by bollwyvl** [comment]
> one of the disadvantages of automerge of course, when dependencies change

Yep, automerge is kinda rough. Things that can help:
- pip check (added)
- download and run the tests (looking into it)
- setting and failing under a coverage threshold

If it gets through all _that_, then I'm usually pretty confident about automerge!

**Comment by bollwyvl** [comment]
@conda-forge/pydata-sphinx-theme back to ready for review, with shiny tests and coverage thresholds to make automerge more confident! (updated description)

**Comment by bollwyvl** [comment]
@conda-forge/pydata-sphinx-theme back to ready for review with 0.6.1!

**Comment by jorisvandenbossche** [comment]
Sory, forgot about this one

File: recipe/meta.yaml [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 517d5f9441d28c64b2bbdb87f4349e5751ba1fe920cd76faa7493688d5f40e6b
+  - folder: dist
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 517d5f9441d28c64b2bbdb87f4349e5751ba1fe920cd76faa7493688d5f40e6b
+  - folder: src
+    url: https://github.com/pydata/pydata-sphinx-theme/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: b7e0df298b71e5f2684744bc58539541d7ca09e03d36f12838a6a2259794d6a8
```
Comment (jorisvandenbossche) [inline recipe/meta.yaml:14]: We could also include the tests in the source distribution for the next release?

**Comment by bollwyvl** [comment]
Hooray tests! I can work up a PR...

**Comment by bollwyvl** [comment]
Up: https://github.com/pydata/pydata-sphinx-theme/pull/393

**Pull request merged**